_assets/patches/geth/0008-tx-pool-nonce.patch

```diff
@@ -0,0 +1,19 @@
+diff --git a/internal/ethapi/api.go b/internal/ethapi/api.go
```

Comment:
hmm, was this file added?

Reply:
That was a conflict after rebase and I couldn't make sense of it. Was it removed the last time?

Reply:
It doesn't seem to exist on `develop`. I'll just remove it then.

Reply:
Yes, I don't see it on the tip of `develop` anymore.